web/post-restoration-transition-application/i18n/locales/en-CA.ts

```diff
@@ -67,5 +68,24 @@ export default {
         TRANP: 'Post Restoration Transition Application'
       }
     }
+  },
+  shares: {
```

Comment:
we're trying to keep this file as simple as possible. I think all of these can just go under 'label' like everything else instead of its own nested json for shares. The only reason we have 'ConnectFeeWidget' like this is because its a connect component (and we're planning to flatten that later)